kafka-ui-react-app/src/components/Brokers/__test__/BrokersContainer.spec.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { containerRendersView } from 'lib/testHelpers';
+import Brokers from 'components/Brokers/Brokers';
+import configureStore from 'redux/store/configureStore';
+import BrokersContainer from 'components/Brokers/BrokersContainer';
+import { shallow } from 'enzyme';
+import { Provider } from 'react-redux';
+
+describe('BrokersContainer', () => {
+  containerRendersView(<BrokersContainer />, Brokers);
+
+  const store = configureStore();
+  it('renders view', () => {
+    const component = shallow(
+      <Provider store={store}>
+        <BrokersContainer />
+      </Provider>
+    );
+
+    expect(component.exists()).toBeTruthy();
```

Comment:
If you take a look at the implementation of `containerRendersView`, you'll see that it does almost exactly what you've written here. So this code might be redundant

Reply:
deleted